portal-api/src/v1/users/sign-in-link.service.login-user.test.js

```diff
@@ -62,7 +62,23 @@ describe('SignInLinkService', () => {
       });
     });
 
-    describe('when the user is not blocked', () => {
+    describe('when the user is disabled', () => {
+      let disabledUser;
+      beforeEach(() => {
+        disabledUser = produce(testUser, (draft) => {
+          draft.disabled = true;
+        });
+        when(userRepository.findById).calledWith(disabledUser._id).mockResolvedValueOnce(disabledUser);
+        when(utils.issueValid2faJWT).calledWith(disabledUser).mockReturnValueOnce(tokenObject);
+        when(userRepository.updateLastLogin).calledWith({ userId: disabledUser._id, sessionIdentifier }).mockResolvedValueOnce(undefined);
```

Comment:
`updateLastLogin` is not a property in `users` collection, should we be checking `signInLinkSendCount` and `signInToken`?